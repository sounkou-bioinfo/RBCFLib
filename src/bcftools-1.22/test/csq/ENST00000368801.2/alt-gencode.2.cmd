{bin}/bcftools csq -f ENST00000368801.2.fa -g ENST00000368801.2.gff alt-gencode.vcf -v 0 -C 1 | {bin}/bcftools query -f'%BCSQ\n' | ../sort-csq -q
