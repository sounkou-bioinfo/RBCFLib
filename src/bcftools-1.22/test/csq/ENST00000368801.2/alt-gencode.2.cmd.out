start_retained|HRNR|ENST00000368801|protein_coding|-|1M>1L|11192T>G
